modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-alter.adoc

```diff
@@ -0,0 +1,81 @@
+= rpk cluster quotas alter
+:description: rpk cluster quotas alter
+
+Add or delete a client quota.
+
+A client quota consists of an entity (to whom the quota is applied) and a quota type (what is being applied).
+
+There are two entity types supported by Redpanda: client ID and client ID prefix. Assigning quotas to default entity types is possible using the `--default` flag.
+
+You can perform a dry run using the `--dry` flag.
+
+== Usage
+
+[,bash]
+----
+rpk cluster quotas alter [flags]
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--add |strings |Key=value quota to add, where the value is a float number (repeatable).
+
+|--default |strings |Entity type for default matching, where type is client-id or client-id-prefix (repeatable).
+
+|--delete |strings |Key of the quota to delete (repeatable).
+
+|--dry |- |Key of the quota to delete (repeatable).
+
+|--format |string |Output format. Possible values: `json`, `yaml`, `text`, `wide`, `help`. Default: `text`.
+
+|-h, --help |- |Help for alter.
+
+|--name |strings |Entity for exact matching. Format type=name where type is client-id or client-id-prefix (repeatable).
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Comment:
```suggestion
|--config |string |Redpanda or `rpk` configuration file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Reply:
Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.